tests/integration/stub_test.ts

```diff
@@ -1,4 +1,5 @@
 import { Rhum } from "../../mod.ts";
+import { Stubed } from "../../src/interfaces.ts";
```

Comment:
like i said, we'll need to have a discussion about the stubs and mocks. im reserving my input until we have a discussion about stubs and mocks again